starlight/templates/profile_views/edit_profile.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% load i18n %}
+
+{% block content %}
+
+<div class="ui center aligned container grid">
+  <div class="six wide column">
+    <h1 class="ui black header">{% trans "Edit Profile: " %} {{ employee }}</h1>
+
+    <button class="ui fluid small blue button" onclick="location.href='{% url 'change_password' id=employee.id  %}'">Change Password</button>
+    <br>
+
+    <form action="." method="post" class="ui large form" id="employee_edit_form">
+        {% csrf_token %}
+        <div class="field">
+          <input name="username" id="username" type="text"  value="{{ employee.username }}">
```

Comment:
These fields should have placeholder text or labels so that we know what they are for